bgc_data/bgc_chemistry_data.sql

```diff
@@ -2,7 +2,13 @@
 --includes metadata
 CREATE MATERIALIZED VIEW bgc_chemistry_data AS
    SELECT
-      bm.*,
+      bm."Project",
+      bm."StationName",
+      bm."TripCode",
+      che.sampledatelocal AS "SampleTime_Local",
```

Comment:
Remember we need to format local times to a string, e.g.
https://github.com/aodn/public-schema/blob/821454fd289d6f9818af9f060ccc125c55d02627/bgc_data/bgc_trip_metadata.sql#L11

Reply:
I seem to remember Geoserver is still able to filter on these string fields as if they were timestamps, but would be good to re-confirm that.

Reply:
done in commit [ce7a967](https://github.com/aodn/public-schema/pull/59/commits/ce7a9671b041df029d470e6c655a6337df20fb7d)